src/libs/ReportUtils.ts

```diff
@@ -3713,7 +3713,7 @@ function getReportName(
     }
     const parentReportActionMessage = ReportActionsUtils.getReportActionMessage(parentReportAction);
 
-    if (parentReportAction?.actionName === CONST.REPORT.ACTIONS.TYPE.SUBMITTED) {
+    if (ReportActionsUtils.isActionOfType(parentReportAction, CONST.REPORT.ACTIONS.TYPE.SUBMITTED)) {
         return getIOUSubmittedMessage(parentReportAction);
```

Comment:
hmm same here??

Reply:
yep - create a thread off of this auto-submitted action, you'll see the report name is the old text

Reply:
fixing in https://github.com/Expensify/App/pull/49909